BootloaderCommonPkg/Library/ContainerLib/ContainerLib.c

```diff
@@ -663,7 +665,11 @@ LoadComponentWithCallback (
     }
 
     if (FeaturePcdGet (PcdVerifiedBootEnabled)) {
-      AuthType = AUTH_TYPE_SHA2_256;
+      if(FixedPcdGet8(PcdCompSignHashAlg) == HASH_TYPE_SHA256) {
+        AuthType = AUTH_TYPE_SHA2_256;
+      } else if (FixedPcdGet8(PcdCompSignHashAlg) == HASH_TYPE_SHA384) {
+        AuthType = AUTH_TYPE_SHA2_384;
+      }

```

Comment:
did you miss a else condition ?  What will be AutoType if either of the condition is true ?

Reply:
Default Signing hash configured is SHA2_256. I would assume PcdCompSignHashAlg would be set to SHA256 by default.
I would add else condition to return as UnSupported,